MakefileConstants.mk

```diff
@@ -9,7 +9,7 @@ BUCKET = cz-$(FEATURE_NAME)
 SEMVER_MAJ_MIN = 1.0
 
 # Source Files
-CFN_TEMPLATES = services/connected_account.yaml
+CFN_TEMPLATES = services/connected_account.yaml services/account_type/cloudtrail_owner.yaml
```

Comment:
I merged the resource_owner PR a few minutes ago. You'll have a conflict with this line from that change you'll have to resolve before merging this PR.

Reply:
The conflict has already been fixed.